src/sql/src/plan.rs

```diff
@@ -850,6 +850,7 @@ pub enum ExplaineeStatement {
         raw_plan: HirRelationExpr,
```

Comment:
Same suggestion like I had in the `parser` struct - maybe it's more appropriate to rename this variant from `Query` to `Select` given that that's the name of the wrapped struct that bundles a `Query` with an optional `AsOf`.

Reply:
Done.